kinto_http/tests/functional.py

```diff
@@ -75,13 +75,13 @@ def test_bucket_modification(self):
         assert 'version' not in bucket['data']
 
     def test_bucket_retrieval_fails_when_not_created(self):
-        self.assertRaises(BucketNotFound, self.client.get_bucket,
```

Comment:
@prabhakar267 Since `BucketNotFound` isn't being used now, you should remove it from imports.